docs/proposals/operator-ci-procedures.md

```diff
@@ -0,0 +1,127 @@
+# Proposal: Update CI procedures with csi-baremetal-operator 
+
+Last updated: 12.04.2021
+
+
+## Abstract
+
+csi-baremetal-operator (https://github.com/dell/csi-baremetal-operator) must be integrated with CI procedures using in development at the moment.
+
+## Proposal
+
+#### csi-baremetal-operator-build
+
+Linting & unity testing & bulding stages
+
+Promoted artifacts:
+
+- csi-baremetal-operator docker image
+
+- csi-baremetal-operator helm chart
+
+- csi-baremetal-CR helm chart
+
+The artifacts have equal version gotten from 'make version'
+
+#### csi-baremetal-build
+
+Linting & unity testing & bulding stages
+
+Promoted artifacts:
+
+- csi-baremetal-controller docker image
+
+- csi-baremetal-node docker image
+
+- csi-baremetal-node-controller docker image
+
+- csi-baremetal-scheduler-extender docker image
+
+- csi-baremetal-scheduler-extender-patcher docker image
+
+- csi-baremetal-##mgr docker image
+
+The artifacts have equal version gotten from 'make version'
+
+#### csi-baremetal-operator-ci
+
+e2e testing stage: deploy csi-baremetal-CR with operator + perform base tests
+
+Versions:
+
+- csi-baremetal-operator helm chart -> ${new_version}
+
+- csi-baremetal-CR helm chart -> ${new_version}
+
+
+- csi-baremetal-operator docker image -> ${new_version}
+
+- csi-baremetal-controller docker image -> green
+
+- csi-baremetal-node docker image -> green
+
+- csi-baremetal-node-controller docker image -> green
+
+- csi-baremetal-scheduler-extender docker image -> green
+
+- csi-baremetal-scheduler-extender-patcher docker image -> green
+
+- csi-baremetal-##mgr docker image -> green
+
+#### csi-baremetal-operator-ci
```

Comment:
Can you clarify why there are two separate `csi-baremetal-operator-ci` with different tests and versions changes?